backend/src/main/java/com/cs471/studentLoanSystem/login/LoginSystem.java

```diff
@@ -0,0 +1,43 @@
+package com.cs471.studentLoanSystem.login;
+
+import java.net.URI;
+
+import com.cs471.studentLoanSystem.roles.StudentInfo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+@RestController
+public class LoginSystem {
+
+    @GetMapping("/login")
+    public String loginForm(Model model) {
```

Comment:
If I am not mistaken, this is for pulling up a form called LoginInfo from an HTML template. I don't see the HTML template in the code? I believe it should be under your resources?

Reply:
I think this will get removed in the final version, it was mostly for testing at the moment. We want the login form to come from the PHP frontend, so the backend will just handle the post request.